src/Android/Avalonia.Android/Platform/SkiaPlatform/TopLevelImpl.cs

```diff
@@ -191,6 +182,29 @@ void ISurfaceHolderCallback.SurfaceChanged(ISurfaceHolder holder, Format format,
 
                 base.SurfaceChanged(holder, format, width, height);
             }
+
+            public sealed override bool OnCheckIsTextEditor()
+            {
+                return true;
+            }
+
+            private Action<EditorInfo> _initEditorInfo;
+
+            public void InitEditorInfo(Action<EditorInfo> init)
+            {
+                _initEditorInfo = init;
+            }
+
+            public sealed override IInputConnection OnCreateInputConnection(EditorInfo outAttrs)
+            {
+                if (_initEditorInfo == null)
```

Comment:
This method is called by Android when it wishes to. Throwing an exception here doesn't really make sense.

Reply:
This metod is called to create `InputConnection` with IME. Exception is needed because we should configure connection from one side from one side, and as a part of documentation from another: `IInitEditorInfo` is not really clear solution, but i do have no idea how to delegate this better...

May be implementing ITextInputMethodImpl by `ViewImpl`?

Reply:
Yap, you were right... in AndroidTestApplication this method was called during startup...